jupyter-book/multi_omics/1_0_scATAC_intro.ipynb

```diff
@@ -0,0 +1,144 @@
+{
```

Comment:
<p>Ideally, we could reuse the paper figure here maybe, but likely the publish rights BS won't allow us to do that.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/theislab/extended-single-cell-best-practices/pull/68/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>